Source/WebCore/css/MediaQueryEvaluator.cpp

```diff
@@ -489,6 +489,17 @@ static bool scanEvaluate(CSSValue* value, const CSSToLengthConversionData&, Fram
     return primitiveValue->valueID() == CSSValueProgressive;
 }
 
+static bool forcedColorsEvaluate(CSSValue* value, const CSSToLengthConversionData&, Frame&, MediaFeaturePrefix)
+{
+    auto* primitiveValue = dynamicDowncast<CSSPrimitiveValue>(value);
+    if (!primitiveValue)
+        return false;
+
+    // On macOS, we don't support "High Contrast" mode,
```

Comment:
`On macOS` -> `On Cocoa platforms`

Also, there is a "High Contrast" mode on macOS and iOS (using the "Increase Contrast" setting), but there is no concept of "forced colors".

Reply:
I will reword the comment, it's not very accurate indeed.

For reference, WK already supports the "increase contrast setting" via the "prefers-contrast" MQ : https://github.com/WebKit/WebKit/blob/e66751caa5cc6a8968a262df6c027f19dcc6c0f2/Source/WebCore/css/MediaQueryEvaluator.cpp#L840